tcms/telemetry/static/telemetry/js/testing-breakdown.js

```diff
@@ -0,0 +1,105 @@
+$(document).ready(() => {
+    $('.selectpicker').selectpicker();
+
+    loadInitialProduct();
+
+    document.getElementById('id_select_product').onchange = updateTestPlanSelect;
+    document.getElementById('id_select_test_plan').onchange = reloadCharts;
+});
+
+function loadInitialProduct() {
+    jsonRPC('Product.filter', {}, data => {
+        updateSelect(data, '#id_select_product', 'id', 'name');
+        updateTestPlanSelect();
+    });
+}
+
+function updateTestPlanSelect() {
```

Comment:
I am changing only Product, not touching anything else. Charts are not reloaded. 

IMO you should control how you filter the test plans based on selected product and at the end of this onchange handler simply render the charts again. This looks like to be the least complex way of doing things.